src/app/Fake.IO.FileSystem/paket.references

```diff
@@ -2,4 +2,5 @@ group netcore
 
 FSharp.Core
 System.Diagnostics.FileVersionInfo
-NETStandard.Library
\ No newline at end of file
+NETStandard.Library
+System.IO.FileSystem.Watcher
```

Comment:
If this is not part of NetStandard.Library we should consider making this a separate module as well, shouldn't we? (On the other side I'm not sure if we should have as many modules as we have now...)